src/Shared/Controls/DebugLogs.xaml

```diff
@@ -0,0 +1,116 @@
+﻿<UserControl
+    x:Class="StrixMusic.Controls.DebugLogs"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:StrixMusic.Controls"
+    xmlns:animations="using:Microsoft.Toolkit.Uwp.UI.Animations"
+	xmlns:color="using:OwlCore.WinUI.Converters.Color"
+	xmlns:controls="using:StrixMusic.Controls"
+	xmlns:core="using:Microsoft.Xaml.Interactions.Core"
+	xmlns:coremodels="using:StrixMusic.Sdk.CoreModels"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+	xmlns:labs="using:CommunityToolkit.Labs.WinUI"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:ui="using:Microsoft.Toolkit.Uwp.UI"
+	xmlns:viewModels="using:StrixMusic.Sdk.ViewModels" xmlns:behaviors="using:OwlCore.WinUI.Behaviors"
+    mc:Ignorable="d"
+    d:DesignHeight="300"
+    d:DesignWidth="400">
+
+    <StackPanel
+		MaxWidth="1600"
+		Margin="20"
+		Spacing="4" VerticalAlignment="Top">
+
+
+        <TextBlock
+            MaxWidth="1000"
+			Margin="1,28,0,4"
+			Style="{StaticResource BodyStrongTextBlockStyle}"
+			Text="Debug Info" />
+
+        <labs:SettingsCard
+			Header="Window Size"
+			IsClickEnabled="True" >
+            <labs:SettingsCard.Content>
+                <TextBlock >
+                    <Run Text="Height(px):"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.WindowHeight}"/>
+                    <Run Text=" Width(px):"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.WindowWidth}"/>
+                </TextBlock>
+            </labs:SettingsCard.Content>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			ActionIcon="{ui:SymbolIcon Symbol=Clear}"
+			Header="Delete User Data"
+			HeaderIcon="{ui:SymbolIcon Symbol=Delete}"
+			IsClickEnabled="True">
+            <interactivity:Interaction.Behaviors>
+                <core:EventTriggerBehavior EventName="Click">
+                    <core:InvokeCommandAction Command="{x:Bind AppRoot.AppDebug.DeleteUserDataCommand, Mode=OneWay}"/>
+                </core:EventTriggerBehavior>
+            </interactivity:Interaction.Behaviors>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			Header="Memory"
+			IsClickEnabled="True" >
+            <labs:SettingsCard.Content>
+                <TextBlock >
```

Comment:
Try this:
- Change Header to "Memory Usage"
- Put the memory limit in the SettingsCard description
- Remove the remaining text in the content. We only need the MemoryUsage binding.